<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>OCC::FolderCreationDialog</class>
    <widget class="QDialog" name="OCC::FolderCreationDialog">
        <property name="windowTitle">
            <string>Create new folder</string>
        </property>
        <layout class="QVBoxLayout">
            <item>
                <widget class="QLineEdit" name="newFolderNameEdit">
                    <property name="placeholderText">
                        <string>Enter folder name</string>
                    </property>
                     <property name="contextMenuPolicy">
                        <enum>Qt::NoContextMenu</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="OCC::sesSnackBar" name="errorSnackbar">
                    <property name="error">
                        <string>Folder already exists</string>
                    </property>
                    <property name="wordWrap">
                        <bool>false</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout">
                    <item>
                        <spacer>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Expanding</enum>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QDialogButtonBox" name="buttonBox">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="standardButtons">
                                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                            </property>
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum" />
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>OCC::FolderCreationDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>OCC::FolderCreationDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>